docs/outlook/faq-nested-app-auth-outlook-legacy-tokens.md

```diff
@@ -165,7 +165,7 @@ If you submit an issue, please include the following information.
 - Version of msal-browser.
 - Logs from msal-browser.
 
-## Developer troubleshooting questions
+## Developer questions
```

Comment:
```suggestion

```
I don't think you need this heading level.

Reply:
It adds it to the top inline TOC which is handy to keep this separate from the admin section.